src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -1060,8 +1060,9 @@ def iot_central_app_create(
               sku=appSku,
               template=template)
 
-    createResult = client.apps.create_or_update(
-        resource_group_name, app_name, app)
+    if no_wait:
+        return client.apps.create_or_update(resource_group_name, app_name, app)
+    createResult = LongRunningOperation(cmd.cli_ctx)(client.apps.create_or_update(resource_group_name, app_name, app))
```

Comment:
How about using `return sdk_no_wait(no_wait, client.apps.create_or_update, resource_group_name=resource_group_name, app_name=app_name, app=app)`

Reply:
@kawayiboy May I ask why you marked it as resolved?

Reply:
> 
> 
> How about using `return sdk_no_wait(no_wait, client.apps.create_or_update, resource_group_name=resource_group_name, app_name=app_name, app=app)`

What's the difference?

Reply:
> What's the difference?

I personally think it's more concise